core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessorState.java

```diff
@@ -51,14 +51,15 @@
 
 public class JsonQueryProcessorState implements Mutable, Closeable {
     public static final String HIDDEN = "hidden";
+    static final int QUERY_SETUP_FIRST_RECORD = 0;
```

Comment:
nit: this file is not formatted, please reformat.

Reply:
done